### PR TITLE
fix: create profile UI

### PR DESCRIPTION
**Pull request by hansac91**

### CODE REVIEW

File: src/components/Dropdowns/DropdownBio/DropdownBio.module.scss [hunk 1]
```diff
@@ -75,14 +75,14 @@
...
-  position: relative;
-  width: 16px;
-  height: 16px;
-  border: 1px solid #e2eaf1;
-  border-radius: 2px;
-  cursor: pointer;
-}
+// .checkbox {
+//   position: relative;
+//   width: 16px;
```
Comment (Sumick) [inline src/components/Dropdowns/DropdownBio/DropdownBio.module.scss:80]: **nitpick:** Comment to remove?

File: src/components/TextInput/TextInput.tsx [hunk 2]
```diff
@@ -49,6 +53,8 @@ const TextInput: React.FC<TextInputProps> = ({
         onChange={handleChange}
         name={name}
         disabled={disabled}
+        onMouseOver={onMouseOver}
+        onMouseOut={onMouseOut}
```
Comment (Sumick) [inline src/components/TextInput/TextInput.tsx:57]: **question:** Why this way? Hover was not working?

**Review by Sumick — APPROVED** [review]
Looks good! Leaving 2 comments

**Review by wawrzonnn — APPROVED** [review]

**Review by mmcode9407 — COMMENTED** [review]
Almost good :) We need some tooltips yet (on locationPreferences, on TechStack and on Bio).

File: src/components/TextInput/TextInput.tsx (hunk 2 above)
Comment (hansac91) [reply src/components/TextInput/TextInput.tsx:57]: Imho I didn't come up to this idea. I've already fixed it to your version ;)

File: src/components/Dropdowns/DropdownBio/DropdownBio.module.scss (hunk 1 above)
Comment (hansac91) [reply src/components/Dropdowns/DropdownBio/DropdownBio.module.scss]: removed

**Review by darionauta — APPROVED** [review]

**Pull request merged**